src/Microsoft.Security.Utilities.Benchmarks/SecretMaskerDetectionBenchmarks.cs

```diff
@@ -91,8 +91,8 @@ private string GeneratePrefix(int length, bool random)
 
             if (random)
             {
-                var byteCount = Base64CharsToBytes(length);
-                var data = new byte[byteCount];
+                int byteCount = Base64CharsToBytes(length);
```

Comment:
>int [](http://example.com/codeflow?start=16&length=3)

this conversion looks right to me, as retval type isn't explicit (though we're not surprised by it)